lib/cisco_node_utils/interface.rb

```diff
@@ -283,7 +289,7 @@ def default_duplex
 
     def negotiate_auto_lookup_string
       case @name
-      when /Ethernet/i
+      when /Ethernet/i, /GigE/i, /MgmtEth/i
```

Comment:
Consider making this search pattern a constant

```
ETHERNET = Regexp.new('(Ethernet|GigE|MgmtEth),  Regexp::IGNORECASE)
```
*I think that's the right syntax*